crates/assistant_tools/src/fetch_tool.rs

```diff
@@ -69,10 +69,9 @@ impl FetchTool {
             .to_str()
             .context("invalid Content-Type header")?;
         let content_type = match content_type {
-            "text/html" => ContentType::Html,
-            "text/plain" => ContentType::Plaintext,
+            "text/html" | "application/xhtml+xml" => ContentType::Html,
             "application/json" => ContentType::Json,
-            _ => ContentType::Html,
```

Comment:
@maxdeviant Figure I should check, is there a motivation for falling back to HTML instead of to plaintext?